core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -700,6 +704,11 @@ public TypeProvider getTypes()
             return types;
         }
 
+        public TaskContext getTaskContext()
```

Comment:
This should be `getPhysicalWrittenDataSize` instead for consistency with other methods here which provide getters without exposing task context directly